scaling_performance/scaling_cluster_monitoring.adoc

```diff
@@ -30,7 +30,7 @@ link:https://www.redhat.com/en/technologies/cloud-computing/openshift-container-
 nodes with non-volatile memory express (NVMe) drives.
 * Use persistent storage when configuring *openshift-monitoring*. Set
 `openshift_cluster_monitoring_operator_prometheus_storage_enabled=true`.
-* Use link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_glusterfs[GlusterFS as storage] on top of NVMe drives.
+* Use link:https://docs.openshift.com/container-platform/3.11/install_config/persistent_storage/persistent_storage_glusterfs.html[GlusterFS as storage] on top of NVMe drives.
```

Comment:
We shouldn't use a link to our own docset. This will break across products, and will be obsolete if another version is ever released. Instead, use an xref to the file so that the link is consistent across versions. I tested the following, and this seems to work:

  xref:../install_config/persistent_storage/persistent_storage_glusterfs.adoc#overview[GlusterFS as storage]

Reply:
@huffmanca, this should be set now.